packages/mml-web/src/elements/Cube.ts

```diff
@@ -58,6 +58,7 @@ export class Cube extends TransformableElement {
     opacity: (instance, newValue) => {
       instance.props.opacity = parseFloatAttribute(newValue, defaultCubeOpacity);
       if (instance.material) {
+        instance.material.transparent = instance.props.opacity === 1 ? false : true;
```

Comment:
My IDE is hinting that we should convert this to `instance.props.opacity !== 1` and I think that makes sense as a simplification.